### ISSUE TITLE
Trailing whitespaces are not removed on save

### ISSUE THREAD

**Issue by jsnjack** [Komodo/KomodoEdit#536]
I have an ansible playbook file in yml format. In Komodo Edit 9.2 I have Clean trailing whitespace and EOL markers option checked (on file save). Komodo does not remove whitespaces on Save but does remove them if I check the menu option Code > Remove trailing whitespaces manually. Removing whitespaces from python files works as expected.

**Comment by Naatan** [comment]
So you're saying it only doesn't happen on yml files?

Could you try creating a brand new yml file and see if it reproduces?

**Comment by jsnjack** [comment]
Yes, I can reproduce it with a new file:
```
- name: Zorro
  copy: banana
                             
- name: New task
  copy: Whitespaces between blocks named Zorro and New task are not removed
```

**Comment by ervumlens** [comment]
YAML has ["significant trailing whitespace."](https://github.com/Komodo/KomodoEdit/blob/8865f7249de2715c648b6aebc37b0f00b6466214/src/python-sitelib/langinfo_doc.py#L166) This is `False` for most language.

The setting prevents Komodo [from cleaning the file](https://github.com/Komodo/KomodoEdit/blob/ea3d0d28c6544861544300eb653d755626c086eb/src/views/koDocument.py#L1695) (note the `not` in the code).

**Comment by jsnjack** [comment]
Hello Nathan, so what is the status of the issue? Is it fixed or it is not possible to fix?





On Thu, Oct 29, 2015 at 8:41 AM -0700, "Nathan Rijksen" <notifications@github.com> wrote:












Closed #536.

**Comment by Naatan** [comment]
Sorry this issue got auto-closed due to age. But it seems like it was right to close it anyway; trailing whitespace trimming is disabled in YML because the language inherently depends on trailing whitespace in its structure. Until whitespace trimming becomes context sensitive there is no better way of handling this.
